mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -158,10 +158,10 @@ CLASS net/minecraft/unmapped/C_smpzexmx net/minecraft/client/gui/DrawableHelper
 			COMMENT the width of the rectangle
 		ARG 7 height
 			COMMENT the height of the rectangle
-		ARG 8 textureHeight
-			COMMENT the height of the entire texture
-		ARG 9 textureWidth
-			COMMENT the width of the entire texture
+		ARG 8 textureWidth
+			COMMENT the width of the entire texture	
+		ARG 9 textureHeight
+			COMMENT the height of the entire texture	
```

Comment:
Could you remove the trailing whitespace here?

Reply:
Done.